projects/arlas-toolkit/src/lib/components/filters/filters.component.html

```diff
@@ -3,7 +3,7 @@
     <mat-basic-chip class="collection-chip" id="arlas-count-{{c.collection}}" [style.borderColor]="c.color"
       *ngIf="!c.ignored">
       <div class="count-container">
-        <div *ngIf="c.hasCentroidPath" class="zoom" (click)="zoomToData(c.collection)"
+        <div *ngIf="showZoomToData" class="zoom" (click)="zoomToData(c.collection)"
```

Comment:
Shouldn't we still also check that there is the required geometry to perform the zoom to avoid errors ?

Reply:
I'm on it, I have to add an input for the type of "zoom to"

Reply:
Done ✅